### PR TITLE
Sort rengo challenges by timesetting first (and undebug ChallengeModal)

### PR DESCRIPTION
**Pull request by GreenAsJade**
Fixes the problem of finding live rengo challenges

## Proposed Changes

Sort rengo challenge list by isLive() (which happens to include Blitz, IIUC)

### CODE REVIEW

**Comment by GreenAsJade** [comment]
Actually, this is not the best answer.  It should be sorting by time_per_move.

**Comment by GreenAsJade** [comment]
Yeah, I think sorted by time_to_move is the most helpful.

File: src/views/Play/Play.tsx [hunk 1]
```diff
@@ -180,15 +180,15 @@ export class Play extends React.Component<{}, PlayState> {
 
             if (C.rengo) {
                 rengo.push(C);
-            } else if (C.time_per_move > 0 && C.time_per_move < 3600) {
+            } else if (C.time_per_move > 0 && C.time_per_move < 3600) { // TBD: why aren't we using isLive here?
```
Comment (anoek) [inline src/views/Play/Play.tsx:183]: likely because it predated isLIve?

Comment (GreenAsJade) [reply src/views/Play/Play.tsx:183]: Makes sense - if we're happy with that, I'll clean next time through

**Pull request merged**